src/common/collection_column_specs/samples-ENIGMA.yml

```diff
@@ -0,0 +1,127 @@
+columns:
+  - key: coll
+    type: string
+    filter_strategy: identity
+    non_visible: true
+
+  - key: load_ver
+    type: string
+    filter_strategy: identity
+    non_visible: true
+
+  - key: _mtchsel
+    type: string
+    filter_strategy: identity
+    non_visible: true
+
+  - key: kbase_ids
+    type: string
+    filter_strategy: identity
+    non_visible: true
+    display_name: Related KBase IDs
+    category: Identifiers
+    description:
+
+  - key: genome_count
+    type: int
+    display_name: Genome Count
+    category: Statistics
+    description:
+
+  - key: kbase_sample_id
+    type: string
+    filter_strategy: identity
+    display_name: KBase Sample ID
+    category: Identifiers
+    description:
+
+  - key: kbase_display_name
+    type: string
+    filter_strategy: ngram
+    display_name: KBase Name
+    category: Identifiers
+    description:
+
+  # sample attributes currently accessible in the ENIGMA collection
+  # parsed using parse_sample_spec in parse_sample_spec.py
+  - key: enigma:collection_time
+    type: string
+    filter_strategy: identity
+    display_name: Collection Time
+    category: Collection
+    description: Collection Time
+  - key: enigma:date
+    type: date
+    display_name: Date
+    category: Collection
+    description: Date (YYYY-MM-DD)
```

Comment:
It there any info as to what the date is for?

Reply:
not from the template. I update the script to put `Sample collection date` as the description.  

```
  Date:
    aliases:
    - date
    category: collection
    definition: Date (YYYY-MM-DD)
    example: '2019-06-18'
    order: 8
    required: false
    transformations:
    - parameters:
      - enigma:date
      transform: map
```

Reply:
do we know it's the collection date or is that a guess?

Reply:
It's just my best guess.

Reply:
Better to leave it ambiguous than possibly wrong in that case

Reply:
👍

Reply:
:+1: